db/migrations/0004_alter_moviesession_cinema_hall_and_more.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-11-01 23:08
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_movie_actors_alter_movie_genres'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='moviesession',
+            name='cinema_hall',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.cinemahall'),

```

Comment:
The `ForeignKey` reference to 'db.cinemahall' should be 'db.CinemaHall' to match the model name defined in the migration `0002_cinemahall_movie_moviesession`.